AWS.md

```diff
@@ -0,0 +1,36 @@
+# AWS Deployment
```

Comment:
The file is double now :-)

Maybe add something to the README.md of the root pointing to `aws/README.md`

Reply:
Fixed